docs/theme-context.rst

```diff
@@ -0,0 +1,121 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the `html_context Sphinx setting`_ in
+the ``conf.py`` file. This extra context is used by the Read the Docs Sphinx Theme
+to add additional features to the built documentation
+
+.. _html_context Sphinx setting: http://www.sphinx-doc.org/en/stable/config.html#confval-html_context
+
+Context injected
+----------------
+
+Here is the full list of values injected by Read the Docs as a Python dictionary.
+Note that this dictionary is injected under the main key `readthedocs`:
+
+
+.. code:: python
+
+    'readthedocs': {
+        'version': {
+            'pk': int,
+            'name': str,
+            'slug': str,
+            'downloads': {
+                'pdf: str,
+                'htmlzip': str,
+                'epub': str
+            },
+            'resource_uri': '/api/v2/version/{pk}/'
```

Comment:
is `resource_uri` a standard term we use? I don't see it in our existing API v2 return values. Also, I think we should put a fully qualified URL (eg. https://readthedocs.org/api/v2/version/{pk}/`

Reply:
I took that name from the APIv1 http://docs.readthedocs.io/en/latest/api.html?highlight=resource_uri#get--api-v1-build-id-

But the standard way seems to be to add a `links` object like:

```
"links": [
        {
            "href": "https://readthedocs.org/api/v2/version/{pk}/",
            "rel": "self"
        }
    ]
```

(I think we shouldn't make it a list, just a dict --but that is not standard)

* https://restfulapi.net/hateoas/
* https://spring.io/understanding/HATEOAS